### PR TITLE
Update overlay image logic following Fastly IO migration

### PR DESCRIPTION
**Pull request by philmcmahon**
## What does this change?
Renames the query string parameters we're using to set overlay images so that they are aligned with the respective names in Fastly IO world - see https://github.com/guardian/fastly-image-service/pull/48

The main change here is that overlay assets are now being served from S3 directly, so are no longer part of the frontend project. This will unfortunately add some friction when we want to update them, but hopefully not too much - any new images just need to be uploaded to `s3://<static bucket>/overlays/`

## What is the value of this and can you measure success?
Simplified vcl and image url generation logic

### Tested

- [ ] Locally
- [x] On CODE (optional)

### CODE REVIEW

**Comment by mchv** [comment]
> The main change here is that overlay assets are now being served from S3 directly, so are no longer part of the frontend project. This will unfortunately add some friction when we want to update them, but hopefully not too much - any new images just need to be uploaded to s3:///overlays/

What is the benefit of serving them from `S3`?

**Comment by philmcmahon** [comment]
We need to serve them from a location that the Fastly IO VCL understands as a backend host.

File: docs/03-dev-howtos/20-update-overlay-images.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Updating Social Media Overlay Images
+====================================
+
+We use the [overlay capability of Fastly IO](https://docs.fastly.com/api/imageopto/overlay) to generate images for social
+media - [here]() is an example with the guardian logo overlayed on it. The code that generates the urls for these images
```
Comment (katebee) [inline docs/03-dev-howtos/20-update-overlay-images.md]: minor: looks like the url is missing inside the link markdown?

major: dev howtos! Thank you thank you thank you for writing docs 👏 👏 👏

**Review by katebee — APPROVED** [review]
👍 💎 This looks good to me!

File: docs/03-dev-howtos/20-update-overlay-images.md (hunk 1 above)
Comment (philmcmahon) [reply docs/03-dev-howtos/20-update-overlay-images.md]: Nice spot, thanks Kate!

**Pull request merged**